.travis.yml

```diff
@@ -5,18 +5,14 @@ go:
 
 install:
   - go get -t ./...
+  - go get github.com/dedis/Coding || true
   - go get golang.org/x/tools/cmd/cover
-  - go get github.com/mattn/goveralls
-
-env:
-  global:
-    secure: "DWzDQe3PqC9KMMQXzWC5lrk0xxFbQdYvKbyygowW/OpMx57m2Npw98WkWA488sVBw1L4IFqnS5XQpagBshGtzG7G4EDZ9M8xZrqjGd963ugiEASeAL1UdmySP3VInDm8J6cEsxxhb9Hy/lcOrMR7E6xKlwx00KNP8iRyJR4lOju8Bnbeg4F5/UsMGxptoFkQeOr2zy3UHkcEP81wv2VrPSMKRKDQPf2rdxo9J5arUpo11LgDCRuMtQtylV9w7eNEUMM5BUYFcPBWu96LCGRSqcSWL789Y2gF99VhpfzCY+nr+m94o6RUZGeiA9DwRqTbFsGrLqxZGNJZ5mV9B33zgjfZ/pRrZahf32UccFWWSrxIuP3+pnWjiW2c4vYsgULLXGKrQbgMPMpFHZWmyqitBTWE3OwKWb8Hp3qZb10CKA9pfJ64nd0NtXAhbMsKswBomExb1bZMlSBu8MpCoEK7qKDYJrHQKWwtSck8AaeEnDQpv7hldZOb7C8kqEhTa/zmtW98Yenrq0P8sT2i7G98ZljbzM4ajIXeWMLmnEBAzgy12RQ/vfNFwmsVwT9t1zVEW1p1REbVKoMZJ+boLv4Soklx2SIscAEn59QEiAOTf65CoWqms3xPZ4eTMJ0AeuL4lOhTP/3hAIJ7VP3Jbl5J8RUaoO58x9MNwhGyRXmg0Ws="
+  - go get github.com/dedis/goveralls
 
 script:
   - make test
 
-after_success:
-  - $GOPATH/bin/goveralls -coverprofile=profile.cov -service=travis-ci -repotoken $COVERALLS_TOKEN
-
 notifications:
   email: false
+
+sudo: required
```

Comment:
Why requiring sudo ? Apparently, on this [line](https://github.com/dedis/onet/pull/225/files#diff-891d77d087977450358304892d384a08L11) you already check if sudo is there or not and act accordingly.

Reply:
True - there are other tests that require `sudo` - should I add the same `t.Skip` there?

Reply:
I don't know. Why do we need sudo at all ? when I run `go test` it uses sudo on my machine ?

Reply:
Because of

https://github.com/dedis/onet/blob/4b2d0e5f1133fc8c1639303c5f025bb55afb5400/simul/platform/localhost.go#L166

but perhaps `sudo` is not really needed there. In fact, you tell me: this was for your pairing-based crypto libraries - can you install them also as the local user?

Reply:
I would prefer to assume the script knows better if it needs sudo or not and if it can run with or without sudo instead of directly forcing him as sudo. 
So I think we can remove the sudo necessary condition here.